import_flow_events.py

```diff
@@ -54,22 +59,63 @@
       utmTerm VARCHAR(40)
     );
 """
+Q_CREATE_METADATA_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_metadata (
+      flowId VARCHAR(64) NOT NULL UNIQUE,
+      beginTime TIMESTAMP NOT NULL SORTKEY,
+      duration INTERVAL NOT NULL,
```

Comment:
*Sigh*, [`INTERVAL` is not supported in redshift](http://docs.aws.amazon.com/redshift/latest/dg/c_unsupported-postgresql-datatypes.html).